pages/common/trufflehog.md

```diff
@@ -0,0 +1,36 @@
+# trufflehog
+
+> Find and verify credentials in files, git repositories, s3 buckets, docker images.
+> More information: <https://github.com/trufflesecurity/trufflehog>.
+
+- Scan a git repo for only verified secrets:
+
+`trufflehog git {{https://github.com/trufflesecurity/test_keys}} --only-verified`
+
+- Scan a GitHub Org for only verified secrets:
+
+`trufflehog github --org={{trufflesecurity}} --only-verified`
+
+- Scan a GitHub Repo for only verified keys and get JSON output:
+
+`trufflehog git https://github.com/trufflesecurity/test_keys --only-verified --json`
+
+- Scan a GitHub Repo along with its Issues and Pull Requests:
+
+`trufflehog github --repo=https://github.com/trufflesecurity/test_keys --issue-comments --pr-comments`
+
+- Scan an S3 bucket for verified keys:
+
+`trufflehog s3 --bucket=<bucket name> --only-verified`
+
+- Scan S3 buckets using IAM Roles:
+
+`trufflehog s3 --role-arn={{iam-role-arn}}`
+
+- Scan individual files or directories:
+
+`trufflehog filesystem {{path/to/file1.txt path/to/file2.txt path/to/dir ...}}`
```

Comment:
```suggestion
`trufflehog filesystem {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
```
We use this argument across our pages to highlight placeholders that accept files and directories.

**Ref.** https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md